startleft/cli.py

```diff
@@ -135,5 +161,18 @@ def search(type, query, filename):
     logger.info("Running JMESPath search query against source files")
     iac_to_otm.search(type, query, filename)
 
+
+@click.option('--server', default='http://localhost:8080', envvar='IRIUS_SERVER',
```

Comment:
We should not provide a default value for this, but should instead fail-fast if the value is not provided. As of now, launching this server gives no information of which IriusRisk instance will be affected by the changes we'll push through the API calls